### PR TITLE
feat(about/index): revamp style of about/index page

### PR DESCRIPTION
**Pull request by josix**
## Types of changes

* **New feature**

## Description
Update styling of the about index page following the spec. 

## Steps to Test This Pull Request
Running `npm run dev` and checkout `http://localhost:3000/2022/zh-hant/about/`

## Expected behavior

localhost_3000_2022_zh-hant_about_

### CODE REVIEW

File: pages/about/index.vue [hunk 1]
```diff
@@ -5,65 +5,49 @@
...
-                    v-for="img in reasonImgs"
-                    :key="img.text"
-                    class="flex flex-col items-center"
-                >
+            <div class="grid grid-cols-2 md:grid-cols-4 mx-2 my-7 md:my-16">
+                <div v-for="(img, i) in introImgs" :key="`intro_img_${i}`">
                     <div
                         class="
-                            w-24
+                            w-32
```
Comment (mattwang44) [inline pages/about/index.vue:25]: nit: consider assign a new class and move these lines to CSS block

File: pages/about/index.vue [hunk 2]
```diff
@@ -92,18 +76,25 @@
                     </template>
                 </i18n>
             </div>
-            <p class="w-full text-xs md:text-base lg:text-base leading-6">
-                {{ $t('activityDesc') }}
-            </p>
-            <ul class="m-0 md:m-12 lg:m-12 text-xs md:text-base lg:text-base">
+            <ul
+                class="
```
Comment (mattwang44) [inline pages/about/index.vue:80]: ditto

File: pages/about/index.vue [hunk 3]
```diff
@@ -92,18 +76,25 @@
...
+                    m-0
+                    md:m-12
+                    lg:m-12
+                    text-xs
+                    md:text-base
+                    lg:text-base
```
Comment (mattwang44) [inline pages/about/index.vue:86]: redundant

```suggestion
```

File: pages/about/index.vue [hunk 4]
```diff
@@ -199,12 +178,16 @@ export default {
...
 <style scoped>
-.h3 {
-    @apply font-serif;
-    color: #d1d1d1;
+.title {
+    @apply font-serif text-primary-100 my-3 text-base md:text-lg lg:text-lg font-bold;
+}
+
+.description {
+    @apply w-full text-xs md:text-base lg:text-base;
```
Comment (mattwang44) [inline pages/about/index.vue:186]: redundant

```suggestion
    @apply w-full text-xs md:text-base;
```

File: pages/about/index.vue [hunk 5]
```diff
@@ -199,12 +178,16 @@ export default {
 </script>
 
 <style scoped>
-.h3 {
-    @apply font-serif;
-    color: #d1d1d1;
+.title {
+    @apply font-serif text-primary-100 my-3 text-base md:text-lg lg:text-lg font-bold;
```
Comment (mattwang44) [inline pages/about/index.vue:182]: redundant
```suggestion
    @apply font-serif text-primary-100 my-3 text-base md:text-lg font-bold;
```

**Review by mattwang44 — CHANGES_REQUESTED** [review]

File: pages/about/index.vue (hunk 2 above)
Comment (josix) [reply pages/about/index.vue]: Fixed on 5c1a9f7ddb9bfb30db36f2ecd8cba5f366ffb755

**Review by mattwang44 — APPROVED** [review]
thx for the beautiful work!

**Pull request merged**